ee/tabby-ui/app/(dashboard)/components/sidebar.tsx

```diff
@@ -61,6 +62,9 @@ export default function Sidebar({ children, className }: SidebarProps) {
               <SidebarButton href="/profile">
                 <IconUser /> Profile
               </SidebarButton>
+              <SidebarButton href="/experiments">
+                <IconSparkles /> Experimental AI
+              </SidebarButton>
```

Comment:
```suggestion
```

Don't display it in side bar, it's developer only page

Reply:
removed